vignettes/SEIRDV.Rmd

```diff
@@ -0,0 +1,215 @@
+---
+title: "The SEIRDV model"
+author: "Ioana Bouros"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{The SEIRV model}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(ggplot2)
+library(glue)
+library(dplyr)
+```
+
+## Introduction
+This document explains the basics of the SEIR model. The model describes
+how populations of susceptible, exposed, infectious, recovered and vaccinated
+individuals evolve over time. Waning immunity is also considered in this model,
+so there is a constant supply of susceptibles in the population and make the model
+more realistic.
+
+## The SEIRDV Model
+The SEIRDV model consists of five ODEs describing five populations of susceptible,
+recovered, infectious, recovered and vaccinated individuals. Susceptible
+individuals ($S$) have not been exposed to the virus. Exposed individuals ($E$) have been
+exposed to the virus and have been infected but are not yet infectious to others; they are 
+in the incubation period. Infectious individuals ($I$) can spread the virus to 
+others.  Recovered individuals ($R$) are no longer
+infectious, but they can return to the susceptible population when they lose their
+immunity. Vaccinated individuals ($R$) cannot become infected and are not ill at
+the time, but as it was the case with the recovered population, their immunity
+is temporarily and can become susceptible again with a certain rate. Births and
+deaths unrelated to the infection are not considered in this model. The ODEs describing
+how the five populations evolve over time are
+$$\frac{\text{d}S}{\text{d}t} = -\beta S I - \nu S + \delta_V V + \delta_R R,$$
+$$\frac{\text{d}E}{\text{d}t} = \beta S I -\kappa E,$$
+$$\frac{\text{d}I}{\text{d}t} = \kappa E - (\gamma + \mu) I,$$
+$$\frac{\text{d}R}{\text{d}t} = \gamma I - \delta_R R,$$
+$$\frac{\text{d}V}{\text{d}t} = \nu S - \delta_V V$$
+where $\beta$, $\kappa$, $\gamma$, $\mu$, $\nu$, $\delta_V$ and $\delta_R$ are
+positive parameters. The rate at which susceptible individuals are infected
+depends on the fraction of the population that is susceptible, the fraction of
+the population that is infectious, and the probability of an infectious individual
+infecting a susceptible individual, $\beta$. This is represented in the
```

Comment:
I doubt this is a precise definition for beta, given that in our model the parameter must also account for the average number of contacts, and we happily allow values of beta greater than 1.

Reply:
I have adapted Solveig's and Hui Jia's vignette to be honest. I agree with your point, so I replaced probability by odds (as in odds ratio), which to me this seems more or less what it is. I have done the same in the SEIRD.Rmd for consistency.